pom.xml

```diff
@@ -73,28 +74,22 @@
             <artifactId>gson</artifactId>
             <version>2.2.4</version>
         </dependency>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.11</version>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>org.powermock</groupId>
             <artifactId>powermock-module-junit4</artifactId>
-            <version>1.4.11</version>
+            <version>1.6.6</version>
```

Comment:
1.4.x and 1.5.x pull in `mockito-all`, which is evil as it masks `hamcrest-core` classes.